Tipster/Tipster.cpp

```diff
@@ -85,6 +101,22 @@ Tipster::MessageReceived(BMessage* msg)
 }
 
 
+//Written by PulkoMandy
+void
+Tipster::OpenURL(BString* url)
+{
+	char *argv[2];
+	BString app;
+
+	app = BROWSER_MIME_WEBPOSITIVE;
```

Comment:
Instead of hard-wiring Web+, please use BRosters FindApp() with the html MIME type to find the user's preferred app and use that ref with the Launch() later.